pkg/sdk/testint/procedures_integration_test.go

```diff
@@ -0,0 +1,541 @@
+package testint
```

Comment:
I would like to see here specific tests for dropping (like in other resources) and it would be nice to call the created procedures (there is another question about call).

Reply:
At least leave a comment about calling (like I have also written in other discussion about supporting call action).